tool/planet/constants.go

```diff
@@ -513,6 +513,9 @@ var KubeletConfig = kubeletconfig.KubeletConfiguration{
 	Authorization: kubeletconfig.KubeletAuthorization{
 		Mode: kubeletconfig.KubeletAuthorizationModeWebhook,
 	},
+	// Allow parallel image pulls to avoid head of line blocking
+	// https://github.com/gravitational/gravity/issues/1167
+	SerializeImagePulls: utils.BoolPtr(false),
```

Comment:
I hope kubelet has some sort of limit/backoff on this internally so we don't destroy nodes with subpar i/o performance.